src/libcrun/linux.c

```diff
@@ -516,12 +516,59 @@ parse_idmapped_mount_option (runtime_spec_schema_config_schema *def, bool is_uid
   return 0;
 }
 
+static char *
+format_mount_mappings (runtime_spec_schema_defs_id_mapping **mappings, size_t mappings_len, size_t *written, bool direct)
+{
+#define MAPPING_FMT_SIZE ("%" PRIu32 " %" PRIu32 " %" PRIu32 "\n")
```

Comment:
Not sure if having this define makes sense any more, since it's only used once.
Removing it will make the code more readable and cleaner.

Reply:
You are right.  I will drop it